components/List.css

```diff
@@ -0,0 +1,75 @@
+.component-list{
+    margin-top: 2em;
+}
+
+.List-wrapper{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 10px;
+}
+
+.List-wrapper div{
+    width: 200px;
+    
+}
+
+ .order-list, .unordered-list{
+     margin-left: 1em;
+ }
+
+.type-A{
+    list-style-type: upper-alpha;
+}
+
+.type-a{
+    list-style-type: lower-alpha;
+}
+
+.upper-roman{
+    list-style-type: upper-roman;
+}
+
+.lower-roman{
+    list-style-type: lower-roman;
+}
+
+.type-circle{
+    list-style-type:circle;
+}
+
+.type-square{
+    list-style-type: square;
+}
+
+.stacked-list-wrapper{
+    display: flex;
+    flex-direction: column; 
+    align-items: center;
+    
+}
+
+.stacked-list-wrapper .list-item{
+    max-width: 400px;
```

Comment:
Yep same, like you've modified from px to em below in docs.css, can change from that here too

Reply:
okay, I'll try to fix all issues in refactor branch. Thank you for your review.